src/main/java/qna/domain/Answer.java

```diff
@@ -1,27 +1,33 @@
 package qna.domain;
 
-import qna.NotFoundException;
-import qna.UnAuthorizedException;
-
-import java.time.LocalDateTime;
 import java.util.Objects;
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
 import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+
+import qna.NotFoundException;
+import qna.UnAuthorizedException;
 
 @Entity
 public class Answer extends BaseEntity {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     
-    private Long writerId;
+    @ManyToOne
```

Comment:
ManyToOne의 기본 로딩 전략은 즉시 로딩입니다. 아래와 같이 설정하면 기본 전략을 지연 로딩으로 설정할 수 있어요 :)

저는 개인적으로 로딩 전략을 설정할 때 기본으로 지연 로딩을 설정하고 즉시 로딩은 꼭 필요한 곳에만 적용해주는 방식으로 사용하고 있어요 :)
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```

Reply:
FetchType.LAZY를 기본전략으로 가져가는게 좋군요! 감사합니다~ 👍🏻👍🏻👍🏻